src/en/mangamo/src/eu/kanade/tachiyomi/extension/en/mangamo/dto/ChapterDto.kt

```diff
@@ -0,0 +1,20 @@
+package eu.kanade.tachiyomi.extension.en.mangamo.dto
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class ChapterDto(
```

Comment:
don't serialize values you are not using

Reply:
Commented out unused fields.

Reply:
I would prefer removing them